packages/accounts/src/account.ts

```diff
@@ -145,6 +151,14 @@ interface ActiveDelegatedStakeBalance {
     total: BN | string;
 }
 
+// by default expire meta transactions 600 blocks after the current block height
+const DEFAULT_META_TRANSACTION_BLOCK_HEIGHT_TTL = 600;
```

Comment:
Is it helpful to have a default in the client? (ideally co-located with other relayer configuration)